composer.json

```diff
@@ -20,9 +20,11 @@
   ],
   "require": {
     "php": ">=5.5",
-    "doctrine/lexer": "^1.0.1"
+    "doctrine/lexer": "^1.0.1",
+    "symfony/polyfill-intl-idn": "^1.10"
   },
   "require-dev": {
+    "ext-intl": "*",
```

Comment:
Is that needed? Do the tests fail with only the polyfill?

Reply:
If they do, we should fix the implementation so they pass, and then remove this line. Otherwise, still remove this line.

Reply:
I understand that by adding polyfill something in the implementation should be changed to use that instead of intl.
For example in https://github.com/egulias/EmailValidator/blob/135444003f7032fa45e3a688737fed4522cb9312/EmailValidator/Validation/SpoofCheckValidation.php#L26

Reply:
SpoofChecker isn't present in the polyfill.

Reply:
Can we ask for this to be implemented in the polyfill. Or, could we polyfill it ourselves in this package?

Reply:
I am insufficiently familiar with how spoofchecker works to advise regarding polyfilling, so I'd imagine the two feasible options are:
1) enquire with symfony re: fleshing out the polyfill
2) fork the spoof check validation out into it's own package that requires `ext-intl:*`, negating the need to polyfill it.

Reply:
I find the 2nd option more appealing as is more in control of the validator. Looks like a bit overkill, having a package only for that, however leveragin the full benefit of the polyfill is interesting.